doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1207,6 +1207,13 @@ POSTed JSON example::
       "email": "pete@mailinator.com"
     }
 
+Change User Identifier
+~~~~~~~~~~~~~~~~~~~~~~
+
+The body of the PUT request is the new identifier for the user. Changes identifier for user in ``AuthenticatedUser``, ``BuiltinUser``, ``AuthenticatedUserLookup`` & ``RoleAssignment``.
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/changeIdentifier/$identifier
```

Comment:
I would much prefer a full curl example showing what the body of the PUT request is supposed to look like.

Reply:
Done!

Reply:
Is this the usual convention for these kinds of apis? Just seems easier to me if it was:
 PUT http://$SERVER/api/admin/authenticatedUsers/changeIdentifier/$oldidentifier/$newidentifier

but I'm not sure if that's atypical.